src/app/core/session/session-service/synced-session.service.spec.ts

```diff
@@ -191,22 +195,6 @@ describe("SyncedSessionService", () => {
     flush();
   }));
 
-  it("Remote succeeds, local fails", fakeAsync(() => {
```

Comment:
I dont know what this was supposed to test. The test case where the local initially fails, the remote succeeds in case of a changed password or new user is covered later. I don't see a practical reason where the local login would always fail.